ex2_system/include/housekeeping/housekeeping_task.h

```diff
@@ -19,7 +19,7 @@
 #ifndef EX2_SYSTEM_HOUSEKEEPING_TASK_H_
 #define EX2_SYSTEM_HOUSEKEEPING_TASK_H_
 
-#define HK_DAEMON_STACK_SIZE 12000
+#define HK_DAEMON_STACK_SIZE 1200
```

Comment:
Do we have a standard stack size? Just wondering if this is bigger or smaller than average, or if there is a reason might need more or less stack.

Reply:
For this one we used uxTaskGetStackHighWaterMark and realized that it was using was less memory than we were allocating. And we were running up against the ceiling on the system (malloc was failing)